### PR TITLE
Add verdict administrator review. Fixes #6062.

### LINKED ISSUE #6062: admin interface for review of flagged packages
We're working on a system to detect malicious uploads (per #4998). It's going to be a pipeline where automated systems run checks, flag packages/projects for deletion/review/ok, etc.

So this issue is for a feature in the admin interface where administrators/moderators (#4011) can review those flagged projects and releases and decide what to accept/reject.

We'll probably also want to use this if/when we implement a mechanism for users to report packages (#3896), and if/when we start automatically checking uploaded packages for compliant metadata and installability (#194).

### PR DESCRIPTION
**Pull request by xmunoz**
- Add new `admin.verdicts.review` endpoint
- Change layout of verdict list and detail view and add forms
- Change sort order of the MalwareChecks, and update the tests

### CODE REVIEW

**Comment by xmunoz** [comment]
Here is a way you can generate verdicts to test this. First open the warehouse shell ` docker-compose run web python -m warehouse shell`. Then run this:

```
from warehouse.malware.models import MalwareCheck, MalwareVerdict
from warehouse.packaging.models import File
check = db.query(MalwareCheck).filter(MalwareCheck.name == "SetupPatternCheck").one()
release_files = db.query(File).limit(10)
for f in release_files:
	db.add(MalwareVerdict(release_file=f, check=check, classification="threat", confidence="medium", message="Just a test"))

db.flush()
db.commit()
```

File: warehouse/admin/views/verdicts.py [hunk 1]
```diff
@@ -61,11 +61,43 @@ def get_verdict(request):
...
+            "verdict": verdict,
+            "classifications": list(VerdictClassification.__members__.keys()),
+        }
 
     raise HTTPNotFound
 
 
+@view_config(
+    route_name="admin.verdicts.review",
+    permission="admin",
```
Comment (ewdurbin) [inline warehouse/admin/views/verdicts.py]: permission="moderator" is probably good here so that we can easily add additional volunteers to review.

Comment (ewdurbin) [reply warehouse/admin/views/verdicts.py]: with that in mind, it might make sense to change the name from `administrator_verdict` to `reviewer_verdict`

**Comment by ewdurbin** [comment]
Really just the thoughts around permissions on the ability to review verdicts, and semantics on the name for the db column. Otherwise, this looks great.

**Pull request merged**